applications/conf/routes

```diff
@@ -51,6 +51,8 @@ GET        /https/optin
 GET        /https/optout                                                        controllers.OptInController.handle(feature = "https", choice = "out")
 GET        /new-header/optin                                                    controllers.OptInController.handle(feature = "header", choice = "in")
 GET        /new-header/optout                                                   controllers.OptInController.handle(feature = "header", choice = "out")
+GET        /gallery/optin                                                       controllers.OptInController.handle(feature = "gallery", choice = "in")
+GET        /gallery/optout                                                      controllers.OptInController.handle(feature = "gallery", choice = "out")
```

Comment:
the above routes are just legacy ones, from now on you just need to use the /opt/ route